ngi_reports/utils/entities.py

```diff
@@ -178,8 +178,14 @@ def populate(self, log, organism_names, **kwargs):
         if proj.get('project_summary',{}).get('all_samples_sequenced'):
             self.dates['all_samples_sequenced'] = proj.get('project_summary',{}).get('all_samples_sequenced')
 
+        if proj.get('order_details'):
+            if proj.get('order_details').get('owner'):
+                self.contact = proj.get('order_details').get('owner').get('email', 'NA')
+            else:
+                self.contact = 'NA'
+        else:
+            self.contact = 'NA'
```

Comment:
You can set the default value of contact to `'NA'` here(https://github.com/NationalGenomicsInfrastructure/ngi_reports/blob/e38917bd2bd86a4922e5982a53ecd84cce87eea2/ngi_reports/utils/entities.py#L111) instead of `''` and you could condense it down to
```suggestion
        self.contact = proj.get('order_details', {}).get('owner', {}).get('email', 'NA')
```